keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/ArtifactInstance.kt

```diff
@@ -10,8 +10,7 @@ import java.time.Instant
  * key fields without which it doesn't make sense for an artifact to exist in Managed Delivery terms. It also adds
  * a couple of keel-specific fields to store artifact metadata.
  */
-// TODO: rename to `Artifact` or `ArtifactInstance`
-data class PublishedArtifact(
+data class ArtifactInstance(
```

Comment:
Even just… `Artifact`?

Reply:
I think historically we avoided just `Artifact` because that's the name of the class in `kork` (which incidentally we also use in `keel`). I don't mind, personally, if others agree.

Reply:
Yeah, that's a reasonable point